docs/users/index.adoc

```diff
@@ -30,3 +30,5 @@ include::bpmn-support.adoc[]
 
 include::architecture/00-index.adoc[]
 
+include::glossary.adoc[]
```

Comment:
I would put this before the bpmn-support.
The bpmn-support paragraph is the 1st one that provides details about the lib. Glossary is general paragraph that reminds all import concepts that we use in the lib.

Reply:
as discussed, we may also add links to the glossory in the rest of the user documentation